### DISCUSSION TITLE
Unlisted addons section in the control panel?

### DISCUSSION THREAD

**Discussion by Sennik** [statamic/cms#5017] [Q&A] [answered]
I updated Statamic to the latest version, installed PHP8.1 to be able to install the codedge/statamic-move-entries add-on. I did so via the CP and all looks fine.

But now in the list of installed add-ons it is in the unlisted addons section. I cannot click it, remove it etc.

The functionallity (move entries to other collections) is also nowhere to be found.

What did I do wrong and what is the purpose of the 'unlisted add-ons' section in the add-ons overview when I cannot do anything with the add-ons listed there?

![afbeelding](https://user-images.githubusercontent.com/5636394/148215595-2f5b0062-fb4d-4496-aa06-7b22d5080b50.png)

**Comment by jasonvarga** [comment]
What does `php please support:details` output?

The unlisted addons section lists addons that are installed that aren't available on the Statamic marketplace.
It's possible that you have a version installed that doesn't match what's on the marketplace, or that the product is misconfigured on the website.

**Reply by Sennik** [reply]
Statamic 3.2.29 Pro
Laravel 8.78.0
PHP 8.1.1
aryehraber/statamic-color-extractor 1.1.2
aryehraber/statamic-font-awesome 2.2.0
codedge/statamic-move-entries 1.1
doublethreedigital/duplicator 2.1.0

**Comment by jasonvarga** [comment, marked as answer]
I've updated the product on the website. There was a version mismatch.

If you run `php please addons:discover` and visit the addons section in your control panel, it should no longer be unlisted.

**Reply by Sennik** [reply]
Hi Jason, it is now, thank you ;-)

But:

It only is usable in list view, nowhere to be found in tree view (took me a minute to find out, there is no explanation whatsoever)

First try resulted in a server error (and also afterwards on the target collection itself):

[2022-01-06 06:50:20] local.ERROR: Undefined array key 1 {"userId":"05c1294e-021e-4816-8c38-48f27dc3aada","exception":"[object] (ErrorException(code: 0): Undefined array key 1 at /var/www/testsite/vendor/statamic/cms/src/Stache/Stores/CollectionEntriesStore.php:38)

**Reply by Sennik** [reply]
Without any further changes now the hole site gives a server 500 error now (cp and site). 'artisan cache:clear'  did not help, 

'php please stache:clear' gives the same error:

 ErrorException

  Undefined array key 1

  at vendor/statamic/cms/src/Stache/Stores/CollectionEntriesStore.php:38
     34▕             $relative = substr($relative, strlen($dir));
     35▕         }
     36▕
     37▕         if (Site::hasMultiple()) {
  ➜  38▕             [$site, $relative] = explode('/', $relative, 2);
     39▕             if (! $this->collection()->sites()->contains($site)) {
     40▕                 return false;
     41▕             }
     42▕         }

      +1 vendor frames
  2   [internal]:0
      Statamic\Stache\Stores\CollectionEntriesStore::getItemFilter()

      +29 vendor frames
  32  please:37
      Illuminate\Foundation\Console\Kernel::handle()

BTW: it is a multisite, 2 sites (Which were working as expected).

**Reply by Sennik** [reply]
Ok, I did not check the collections' physical content thoroughly enough. The moved page landed in the target collections root directory while it should have landed in one of the folders of the multisite configuration. When I manually moved it to the 'default'  folder it (cp and site) worked again.

I think it is save to say that this plugin is not multisite compatible. That's a pity because it can be a very usable add-on.

**What it should be capable of (IMHO)**

- Move from list, tree and detailview (now only listview)
- Move to any collection's site (multisite) with the same site as the source as default (but selectable, if possible multiple sites)

**Nice to have**

- preserve or renew creation date
- preserver or renew author

**Reply by duncanmcclean** [reply]
You'd be best opening an issue on the [addon's repository](https://github.com/codedge/statamic-move-entries/issues), as we can't do anything about it here.

**Reply by Sennik** [reply]
I know, I just wrote it here for completeness and others who end up here ;-)

**Reply by jasonvarga** [reply]
I was just explaining why it was showing in the unlisted section.
As for issues about the addon itself, yes, contact the developer.

**Reply by Sennik** [reply]
> I was just explaining why it was showing in the unlisted section. As for issues about the addon itself, yes, contact the developer.

[I already did ](https://github.com/codedge/statamic-move-entries/issues/103);-)

**Comment by Safemood** [comment]
Hi guys 

I'm having the same problem "**UNLISTED ADDONS**"

there are some details

```php
php please support:details
```
```
Statamic 3.2.37 Pro
Laravel 8.83.4
PHP 8.1.3
safemood/statamic-rollback 1.0.1
```
I did run 
```php 
php please addons:discover
```

```
Discovered Addon: safemood/statamic-rollback
Addon manifest generated successfully.
```

but still, having the same problem?

Is there any solution?

**Reply by duncanmcclean** [reply]
I wonder if it has to do with the fact it says 'Supported: No' in the Marketplace for [this addon](https://statamic.com/addons/kbouzidi/statamic-rollback)... 

![image](https://user-images.githubusercontent.com/19637309/157984995-7443b62f-15c0-496c-b66c-e92f69f3640d.png)

**Reply by Safemood** [reply]
thanks for replying 
I guess not  
supported mean if you will provide support for the addons or not 
check this it has no support
https://statamic.com/addons/double-three-digital/duplicator

**Reply by duncanmcclean** [reply]
Oh, that's true... Not sure what the issue is then. It's possible there's an incorrect version somewhere in the Marketplace that's causing the issue.

**Reply by jasonvarga** [reply]
Seems to work fine for me. Does it still show under unlisted? Try clearing your cache.

![image](https://user-images.githubusercontent.com/105211/158201562-218d526a-6f2b-437c-a85e-8e827a83a139.png)

**Reply by Safemood** [reply]
@jasonvarga thnx for sharing that 
for me, it appears under "Unlisted addons" only in the Statamic Pro version otherwise it works as intended.

**Comment by Safemood** [comment]
I'm still trying to figure it out 
thanks anyway man
